### ISSUE TITLE
this #define SHM_START_ADDR may be wrong

### ISSUE THREAD

**Issue by zbhhbzzbh** [ut-osa/strata#14]
this define SHM_START_ADDR  may overlap if one wants to use more dax0 bigger than 36GB!

**Comment by fzheart** [comment]
You are correct. Thanks for pointing this problem. We will check the issue. 
As a temporary solution, I would like to recommend change SHM_START_ADDR on your environment; there is no restriction of SHM_START_ADDR virtual address.
